cranelift/test-zk.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
```

Comment:
One thing that I typically suggest doing is to `set -eux` and `set -o pipefail`. These provide a much better error handling and debugging opportunities in a black box environment like the CI.